api/v1alpha6/types_conversion.go

```diff
@@ -510,6 +509,21 @@ func Convert_v1beta1_OpenStackIdentityReference_To_v1alpha6_OpenStackIdentityRef
 	return nil
 }
 
+/* APIServerLoadBalancer */
+
+func Convert_v1beta1_APIServerLoadBalancer_To_v1alpha6_APIServerLoadBalancer(in *infrav1.APIServerLoadBalancer, out *APIServerLoadBalancer, s apiconversion.Scope) error {
+	return autoConvert_v1beta1_APIServerLoadBalancer_To_v1alpha6_APIServerLoadBalancer(in, out, s)
+}
+
+func restorev1beta1APIServerLoadBalancer(previous *infrav1.APIServerLoadBalancer, dst *infrav1.APIServerLoadBalancer) {
```

Comment:
Please can you add the new restore function to https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/6c83bc47cf44f8222cedbb7801be0e3e734dec64/api/v1alpha6/conversion_test.go#L767 so it will be fuzzed?